tide_core.module

```diff
@@ -143,6 +143,21 @@ function tide_core_form_alter(&$form, FormStateInterface $form_state, $form_id)
       }
     }
   }
+  if ($form_id === 'menu_edit_form') {
+    if (isset($form_state->getUserInput()['op'])) {
+      if ($form_state->getUserInput()['op'] === 'Save') {
+        $form['actions']['submit']['#submit'][] = 'tide_core_menu_submit_handler';
+      }
+    }
+  }
+}
+
+/**
+ * Redirect form upon saving to the Menu UI Module collection.
+ */
+function tide_core_menu_submit_handler(&$form, FormStateInterface $form_state) {
```

Comment:
hey @mayurngondhkar please rename it as `_tide_core_menu_submit_handler`

Reply:
Resolved